common/source/docs/common-gps-for-yaw.rst

```diff
@@ -69,8 +68,8 @@ the two GPS modules to go via the autopilot board.
 
 .. note:: Do not use :ref:`GPS_AUTO_SWITCH<GPS_AUTO_SWITCH>` = 2 (Blend) when using Moving Baseline configurations.
 
-Internal Moving Baseline Systems
-================================
+Single Unit Internal Moving Baseline Systems
+============================================
 
 Some vehicle GPS provide GPS for Yaw utilizing a completely internal dual gps unit and managing the inter gps communication totally internally, rather than having ArduPilot pass data between the GPSes. Examples of these system are the `Blicube GRTK <https://wiki.blicube.com/grtk/>`__ and the `Holybro UM982 <https://holybro.com/products/h-rtk-unicore-um982>`_.
```

Comment:
Here need to add CUAV C-RTK2 HP

Reply:
I will just remove the line... it's unnecessary really and another thing to maintain as devices are added..thanks...will merge later